tldw_Server_API/app/core/AuthNZ/settings.py

```
@@ -982,6 +1072,20 @@ def parse_openai_oauth_enabled(cls, v):
                 return _bool_from_str(env_val)
         return v
 
+    @field_validator(
+        "AUTH_FEDERATION_ENABLED",
+        "MCP_CREDENTIAL_BROKER_ENABLED",
+        "SECRET_BACKENDS_ENABLED",
+        mode="before",
+    )
+    @classmethod
+    def parse_enterprise_feature_flags(cls, v):
```

Ruff (0.15.6)

[warning] Missing return type annotation for classmethod `parse_enterprise_feature_flags`

(ANN206)